brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-3207 Resolved an issue of Timeline Icons look big in customizer preview.

### Description

- Timeline Icon look big issue

### Screenshots

- https://d.pr/v/xkVRQP

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to Astra Customizer
- Select Header builder
- Use widget and drag and drop timeline
- Make sure not big icon issue visible

### Checklist:
